cea/optimization/ntwOpt/NtwMain.py

```diff
@@ -9,7 +9,7 @@
 
 class ntwFeatures(object):
```

Comment:
class names are CamelCase (not camelCase). use `NtwFeatures` instead. In fact, you can use as many characters as you like, so `NetworkFeatures` is even better!

Also, module names are all lower-case and `_` separated words (but try to find single-word options)

`ntwOpt` is also not a good name for a package. Since the package name starts with `cea.optimization`, I think we can just rename it to `network`, right? `cea.optimization.network`.

Unless `ntw` doesn't stand for "network", but then, well, then we have other problems...

I guess this code is inherited from the master students thuy-an and tim, right? We'll just have to clean up the mess now...